index.html

```diff
@@ -9,8 +9,6 @@
 
   <!-- Favicons display -->
   <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png?v=2.0.22">
-  <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png?v=2.0.22">
-  <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png?v=2.0.22">
   <link rel="icon" type="image/svg+xml" href="/favicon/favicon.svg">
   <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg?v=2.0.22" color="#444444">
   <link rel="shortcut icon" href="/favicon/favicon.ico?v=2.0.22">
```

Comment:
Why we not replace all of them?

Reply:
i didn't dug very deep into it, maybe it'll add some additional compatibility 
![image](https://user-images.githubusercontent.com/50058726/207050140-10a77dd8-ce2a-4a6c-9659-fbb5ec07d06e.png)